pyprql/pandas_magic/__init__.py

```diff
@@ -9,4 +9,4 @@
 results_df = df.prql.query('from df | select [age,name,occupation] | filter age > 21')
 
 """
-import pyprql.pandas.prql  # noqa: F401
+import pyprql.pandas_magic.prql  # noqa: F401
```

Comment:
FYI if you come back to this, I would might call it `pandas_accessor` or similar, `magic` is the technical name for the `%%` functionality in jupyter / ipython. But no biggie

Reply:
Ahh , will do.